pkg/machinery/marker.go

```diff
@@ -33,16 +33,27 @@ var commentsByExt = map[string]string{
 
 // Marker represents a machine-readable comment that will be used for scaffolding purposes
 type Marker struct {
+	prefix  string
 	comment string
 	value   string
 }
 
 // NewMarkerFor creates a new marker customized for the specific file
```

Comment:
I think we need to change the comment here to make clear that will create the marker with kbPrefix

Reply:
Added some more details on what prefix is used